gulpfile.js

```diff
@@ -170,3 +169,7 @@ gulp.task('prefetch', function(cb) {
   }
   artifactType = "-bundle";
 });
+
+gulp.task('default', function() {

```

Comment:
This is what I asked for in my comment

    gulp.task('default', ['run']);  // nothing more is necessary

Transpile creates main/index.js, it has nothing to do with generate